src/promptflow/tests/sdk_cli_azure_test/e2etests/test_run_operations.py

```diff
@@ -1226,3 +1226,30 @@ def submit(*args, **kwargs):
                     params_override=[{"name": run_id}],
                 )
                 pf.runs.create_or_update(run=run)
+
+    @pytest.mark.skipif(
+        condition=not is_live(),
+        reason="Seems flow.dag.yaml's upload request's disappeared.",
```

Comment:
@zhengfeiwang  could you help to check the request missing issue?

Reply:
just pushed a commit, and seems it works: <https://github.com/microsoft/promptflow/actions/runs/8261998331?pr=2334>

here is my understanding/guessing:

according to your recording, there should be some other files under `.promptflow`, this will somehow influence the replay test: I understand/guess azure storage sdk will upload flow directory with multi processes, vcrpy patch may not take effect in these new processes, so some requests might be lost.

my workaround is to keep the flow directory .promptflow for the test clean - that's what happening when i checkout to your branch - then record the test, this should work, according to the experiment.